tests/playbooks/tests_wireless_wpa3_owe.yml

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: wlan0
+  tasks:
+    - name: "INIT: wireless tests"
+      debug:
+        msg: "##################################################"
+    - include_tasks: tasks/setup_mock_wifi_wpa3_owe.yml
```

Comment:
Can you explain why this test case pass on RHEL where WPA3 hostapd is not supported?

Reply:
Explained here, https://github.com/linux-system-roles/network/pull/356/files#diff-10f9980f9bfb699b1749b71620e0ec4fa6256bfcbeb8688888f5233e2ed4714bR10

Reply:
That task is only for CentOS 8. RHEL 8 does not match the conditions defined in `when`.

Reply:
OK. I missed the conditions in `tests/tests_wireless_wpa3_owe_nm.yml `